CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guidelines
+If you're interested in contributing to this project, here are a few ways to do so:
+
+### Bug fixes
```

Comment:
is it intentional to go from `#` straight to `###`?

Reply:
Nope I was just to eager :D put them back to `##`